webdriver/tests/bidi/browsing_context/set_viewport/set_viewport.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+
+from ... import get_viewport_dimensions
+
+
+@pytest.mark.asyncio
+async def test_set_viewport(bidi_session, new_tab):
```

Comment:
If you don't mind maybe you can create one further test which sets the viewport to the exact same dimensions? I noticed that I had a bug in our Firefox implementation which caused the command to hang because it was waiting for a resize of the viewport.

Reply:
Added